libs/output-mapping/src/Writer/TableWriter.php

```diff
@@ -84,11 +87,24 @@ public function uploadTables(
 
         $strategy = $this->strategyFactory->getTableOutputStrategy($stagingStorageOutput, $isFailedJob);
 
-        $mappingSources = $strategy->getMappingResolver()->resolveMappingSources(
-            $sourcePathPrefix,
-            $configuration,
-            $isFailedJob,
-        );
+        $mappingResolver = $strategy->getMappingResolver();
+        if ($mappingResolver instanceof LocalMappingResolver
+            && $this->clientWrapper->getToken()->hasFeature(self::OUTPUT_MAPPING_SLICE_FEATURE)
+        ) {
+            $mappingSources = (new SliceHelper($this->logger))->sliceSources(
```

Comment:
https://github.com/keboola/platform-libraries/pull/184/files#diff-9f8440c592138fd90a248ea5c37226fd967a58c0593ddb5ed789af5e838a24e7R43

Reply:
Máš v tom zmatek. Já ten původni PR přejmenoval tohle je nový. Na tohle jsem ještě review nechtěl a není to hotový.

Reply:
Mam to tu rozdelany, pujde to dovnitr. Tohle byla pracovni verze.